_rules/image-not-in-acc-tree-is-decorative-e88epe.md

```diff
@@ -0,0 +1,318 @@
+```
+
+### Failed
+
+#### Failed Example 1
+
+This `img` element with an empty (`""`) `alt` is not [purely decorative][].
+
+```html
+<img src="/test-assets/shared/w3c-logo.png" alt="" />
+```
+
+#### Failed Example 2
+
+This `img` element which is not [included in the accessibility tree][] because `aria-hidden` is set to `true` is not [purely decorative][].
+
+```html
+<img src="/test-assets/shared/w3c-logo.png" aria-hidden="true" alt="W3C logo" />
+```
+
+#### Failed Example 3
+
+This `img` element which is not [included in the accessibility tree][] because it has an [explicit semantic role][] of `none` is not [purely decorative][].
+
+```html
+<img src="/test-assets/shared/w3c-logo.png" role="none" alt="W3C logo" />
+```
+
+#### Failed Example 4
+
+This `svg` element which has a [semantic role][] of `graphics-document` and an empty (`""`) [accessible name][] is not [purely decorative][].
+
+```html
+<p>Best W3C logo:</p>
+<svg viewBox="0 0 512 512">
+	<path
+		d="M108.4 0h23v22.8h21.2V0h23v69h-23V46h-21v23h-23.2M206 23h-20.3V0h63.7v23H229v46h-23M259.5 0h24.1l14.8 24.3L313.2 0h24.1v69h-23V34.8l-16.1 24.8l-16.1-24.8v34.2h-22.6M348.7 0h23v46.2h32.6V69h-55.6"
+	/>
+	<path fill="#e44d26" d="M107.6 471l-33-370.4h362.8l-33 370.2L255.7 512" />
+	<path fill="#f16529" d="M256 480.5V131H404.3L376 447" />
+	<path
+		fill="#ebebeb"
+		d="M142 176.3h114v45.4h-64.2l4.2 46.5h60v45.3H154.4M156.4 336.3H202l3.2 36.3 50.8 13.6v47.4l-93.2-26"
+	/>
+	<path fill="#fff" d="M369.6 176.3H255.8v45.4h109.6M361.3 268.2H255.8v45.4h56l-5.3 59-50.7 13.6v47.2l93-25.8" />
+</svg>
+```
+
+#### Failed Example 5
+
+This `canvas` element which has no [semantic role][] and an empty (`""`) [accessible name][] is not [purely decorative][].
+
+```html
+<canvas id="w3c" width="200" height="60"></canvas>
+<script>
+	const ctx = document.querySelector('#w3c').getContext('2d')
+	ctx.font = '30px Arial'
+	ctx.fillText('ACT Rules!', 20, 40)
+</script>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+This `img` element is [included in the accessibility tree][] because the `alt` attribute is not empty (`""`).
+
+```html
+<img src="/test-assets/shared/w3c-logo.png" alt="W3C" />
+```
+
+#### Inapplicable Example 2
+
+This `img` element is neither [visible][] nor [included in the accessibility tree][].
+
+```html
+<img src="/test-assets/shared/w3c-logo.png" style="display:none" alt="" />
+```
+
+#### Inapplicable Example 3
+
+This `img` element is not [visible][] because it is positioned off screen.
+
+```html
+<style>
+	img {
+		position: absolute;
+		top: -9999em;
+	}
+</style>
+<img src="/test-assets/shared/fireworks.jpg" alt="" />
+```
+
+#### Inapplicable Example 4
+
+This `svg` element is ignored because it is a child of a link that provides its [accessible name][].
+
+```html
+<a href="//example.org" aria-label="SVG star">
+	<svg height="200" xmlns="http://www.w3.org/2000/svg">
+		<polygon points="100,10 40,180 190,60 10,60 160,180" fill="yellow" />
+	</svg>
+</a>
+```
```

Comment:
Currently, this is applicable.
I guess the **named from author** part of the applicability was meant to **exclude** stiff, not to **include** them.

Reply:
Fixed the applicability. Please check again.